test/quantization/test_quantized_op.py

```diff
@@ -118,68 +118,164 @@ def _get_random_tensor_and_q_params(shapes, rand_scale, torch_type):
 
 class TestQuantizedOps(TestCase):
 
-    """Tests the correctness of the quantized::relu op."""
-    @given(X=hu.tensor(shapes=hu.array_shapes(1, 5, 1, 5),
-                       qparams=hu.qparams()))
-    def test_qrelu(self, X):
+    # When writing a unit test for the activation function,
```

Comment:
nit: use doc string for this comment? and move this to function `_test_activation_function`?